pkgs/applications/gis/open-orienteering-mapper/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchurl, gdal, cmake, proj, clipper, zlib, qtbase, qttools
+  , qtlocation, qtsensors, doxygen, cups, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  name = "OpenOrienteering-Mapper-${version}";
+  version = "0.8.1";
+
+  buildInputs = [ gdal qtbase qttools qtlocation qtsensors clipper zlib proj doxygen cups];
+
+  nativeBuildInputs = [ cmake makeWrapper ];
+
+  enableParallelBuilding = true;
+
+  # Don't build the manual or bundle licenses
+  patches = [ ./license.patch ];
```

Comment:
You can use `-DLICENSING_PROVIDER:BOOL=OFF` https://github.com/OpenOrienteering/mapper/issues/886#issuecomment-308337508